### PR TITLE
uftp 4.9 (new formula)

### PR DESCRIPTION
**Pull request by nlitsme**
### All Submissions:

- [*] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew/blob/master/.github/CONTRIBUTING.md) document?
- [*] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew/pulls) for the same update/change?

_You can erase any parts of this template not applicable to your Pull Request._

### New Formulae Submissions:

- [*] Does your submission pass
`brew audit --strict --online <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [*] Have you built your formula locally prior to submission with `brew install <formula>`?

### Changes to Homebrew's Core:

- [*] Have you added an explanation of what your changes do and why you'd like us to include them?
- [*] Have you written new tests for your core changes, as applicable? [Here's an example](https://github.com/Homebrew/homebrew/pull/49031) if you'd like one.
- [*] Have you successfully ran `brew tests` with your changes locally?

UFTP is a tool for Encrypted multicast file transfer.

### CODE REVIEW

**Comment by apjanke** [comment]
@BrewTestBot test this please

Failure was a transient download error.

**Comment by apjanke** [comment]
The Makefile for `uftp` doesn't support `PREFIX`. That's a pretty basic Makefile feature. Would be worth submitting that as an upstream feature request/bug report.

File: Library/Formula/uftp.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+class Uftp < Formula
+  desc "Encrypted multicast file transfer."
+  homepage "http://uftp-multicast.sourceforge.net/"
+  url "https://downloads.sourceforge.net/project/uftp-multicast/source-tar/uftp-4.9.tar.gz"
+  sha256 "5ed7332ec72bc6d0b19d3045ee432bf29d427389d0b711c306fa41ce2cb963e6"
+
+  depends_on "openssl"
+
+  def install
+    system "sed", "-i.bak", "s_/usr/_/_g", "makefile"
```
Comment (DomT4) [inline Library/Formula/uftp.rb]: We should be able to do this with `inreplace` I'd imagine.

File: Library/Formula/uftp.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    system "make", "install",
+                   "DESTDIR=#{prefix}",
+                   "OPENSSL=#{Formula["openssl"].opt_prefix}"
+  end
+
+  test do
+    assert_match /list_file/, shell_output("#{bin}/uftp -h 2>&1", 1)
+    assert_match /list_file/, shell_output("#{sbin}/uftpd -h 2>&1", 1)
+
+    genout= shell_output("#{bin}/uftp_keymgt -g ec:secp521r1  test-ec.key 2>&1")
```
Comment (DomT4) [inline Library/Formula/uftp.rb]: Please leave a space between variable `=` and output, i.e.

```ruby
genout = shell_output("#{bin}/uftp_keymgt -g ec:secp521r1  test-ec.key 2>&1")
```

**Comment by apjanke** [comment]
Closing this due to inactivity/lack of response. If you'd still like to pursue this, feel free to respond to the previous comments/questions and re-open on the new [`homebrew/core` repo](https://github.com/homebrew/homebrew-core).

**Pull request closed without merging**